packages/esm-home-app/src/homepage-header/homepage-header.component.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Location } from '@carbon/react/icons';
+import { ConfigObject, useConfig, useSession } from '@openmrs/esm-framework';
+import { DatePicker, DatePickerInput } from '@carbon/react';
+import dayjs from 'dayjs';
+import styles from './homepage-header.scss';
+import HomepageIllustration from './homepage-illustration.component';
+
+interface HomepageHeaderProps {
+  dashboardTitle: string;
+}
+
+const HomepageHeader: React.FC<HomepageHeaderProps> = ({ dashboardTitle }) => {
+  const { t } = useTranslation();
+  const session = useSession();
+  const location = session?.sessionLocation?.display;
```

Comment:
Shouldn't the location maybe _start_ with this but be user-selectable? Otherwise, why are we rendering it at all?

Reply:
Similar to the datepicker comment, for simplicity sake to begin with, the idea was to strip functionality from most of the header and improve over time.